ui/desktop/app/templates/scopes/scope/projects/sessions/index.hbs

```diff
@@ -5,54 +5,60 @@
 
 {{#if (or @model.sessions @model.allSessions)}}
   {{#if @model.isClientDaemonRunning}}
-    <Hds::SegmentedGroup as |S|>
-      <S.Generic>
-        <Dropdown
-          @name={{t 'resources.target.title'}}
-          @itemOptions={{this.availableTargets}}
-          @checkedItems={{this.targets}}
-          @applyFilter={{fn this.applyFilter 'targets'}}
-          @isSearchable={{true}}
-          @listPosition='bottom-left'
-        />
-      </S.Generic>
-      <S.Generic>
-        <Dropdown
-          @name={{t 'form.status.label'}}
-          @itemOptions={{this.sessionStatusOptions}}
-          @checkedItems={{this.status}}
-          @applyFilter={{fn this.applyFilter 'status'}}
-        />
-      </S.Generic>
-      <S.Generic>
-        <Dropdown
-          @name={{t 'resources.scope.title'}}
-          @itemOptions={{this.availableScopes}}
-          @checkedItems={{this.scopes}}
-          @applyFilter={{fn this.applyFilter 'scopes'}}
-          @isSearchable={{true}}
-          as |FD selectItem itemOptions|
-        >
-          {{#let (group-by itemOptions 'scopeModel') as |orgs|}}
-            {{#each orgs as |org|}}
-              <FD.Generic>
-                <FlightIcon @name='org' />
-                {{org.key.displayName}}
-              </FD.Generic>
-              {{#each org.items as |project|}}
-                <FD.Checkbox
-                  @value={{project.id}}
-                  checked={{includes project.id this.scopes}}
-                  {{on 'change' selectItem}}
-                >
-                  {{project.displayName}}
-                </FD.Checkbox>
+    <div class='search-filtering-toolbar'>
```

Comment:
looks like we aren't using this classname?

Reply:
This is the `&-toolbar` class I added

Reply:
ooh ok, it was not obvious at first look, any issues in using the full name>

Reply:
Nope but we also use `&-` a lot, we should probably standardize our conventions for this at some point